src/components/ebay-calendar/calendar.stories.js

```diff
@@ -0,0 +1,150 @@
+import Readme from './README.md';
```

Comment:
This is throwing an error. I did an `npm i` before running. 
<img width="1475" alt="image" src="https://user-images.githubusercontent.com/1675667/225136730-3311a3e7-dc4e-4e37-a0b3-9ca795d1897d.png">

Reply:
Weird, it looks like moving functions out into a helper file didn't work. I moved everything back into `component.js`, it should be working now.